docs/en/03_Upgrading/index.md

```diff
@@ -1,22 +1,25 @@
 title: Upgrading
 introduction: Keep your SilverStripe installations up to date with the latest fixes, security patches and new features.
 
-# Upgrading
+# Upgrading to SilverStripe 3.2
 
-SilverStripe Applications should be kept up to date with the latest security releases. Usually an update or upgrade to 
-your SilverStripe installation means overwriting files, flushing the cache and updating your database-schema. 
+SilverStripe applications should be kept up to date with the latest security releases. Usually an update or upgrade to your SilverStripe installation means overwriting files, flushing the cache and updating your database-schema. 
 
 <div class="info" markdown="1">
-See our [upgrade notes and changelogs](/changelogs) for release-specific information.
+See our [upgrade notes and changelogs](/changelogs/3.2.0) for 3.2.0 specific information, bugfixes and API changes.
 </div>
 
 ## Composer 
 
-For projects managed through Composer, check the version defined in your `composer.json` file. Update the version 
-constraints if required and update composer.
+For projects managed through Composer, update the version number of `framework` and `cms` to `^3.2` in your `composer.json` file and run `composer update`. 
 
-	:::bash
-	composer update
+```json
+	"require": {
+		"silverstripe/framework": "^3.2",
```

Comment:
This is risky in that it would automatically update your project to 3.3 on composer update. Did you mean ~3.2.0?